app/src/main/res/values/colors.xml

```diff
@@ -129,6 +130,7 @@
     <color name="prompt_login_edit_text_cursor_color_dark_theme">#9059ff</color>
     <color name="search_suggestion_indicator_icon_color_dark_theme">#2ac3a2</color>
     <color name="search_suggestion_indicator_icon_bookmark_color_dark_theme">#0090ed</color>
+    <color name="preference_section_header_dark_theme">#ab71ff</color>
```

Comment:
Can you also check if there's a color variable for this one? Thanks!

Reply:
Just pushed a commit for that